src/App/Services/PushNotificationListenerService.cs

```diff
@@ -245,6 +245,30 @@ public async Task OnNotificationTapped(BaseNotificationData data)
             }
         }
 
+        public async Task OnNotificationDismissed(BaseNotificationData data)
+        {
+            Resolve();
```

Comment:
Since you're here, can you remove the `Resolve()` method and use `LazyResolve<>` instead for the services?

Reply:
As far as I see this has not been addressed @andrebispo5

Reply:
sorry i though it was on the `NotificationDismissReceiver` class didn't noticed it was in another file. on it!